public/app/features/dashboard-scene/serialization/transformSceneToSaveModel.test.ts

```diff
@@ -277,6 +277,39 @@ describe('transformSceneToSaveModel', () => {
       expect(saveModel.gridPos?.w).toBe(12);
       expect(saveModel.gridPos?.h).toBe(8);
     });
+    it('Given panel with links', () => {
+      const gridItem = buildGridItemFromPanelSchema({
+        title: '',
+        type: 'text-plugin-34',
+        gridPos: { x: 1, y: 2, w: 12, h: 8 },
+        links: [
+          // @ts-expect-error Panel link is wrongly typed as DashboardLink
```

Comment:
Adding those annotations as it seems like panel links are incorrectly typed in the schema as DashboardLinks, while they should be DataLinks instead. But will correct the schema in the followup PR.